src/main/java/portablejim/bbw/compat/architecturecraft/ArchitectureCraftCustomMapping.java

```diff
@@ -0,0 +1,31 @@
+package portablejim.bbw.compat.architecturecraft;
+
+import net.minecraft.block.Block;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.tileentity.TileEntity;
+
+import cpw.mods.fml.common.registry.GameRegistry;
+import gcewing.architecture.common.tile.TileShape;
+import portablejim.bbw.basics.Point3d;
+import portablejim.bbw.core.conversion.CustomMapping;
+import portablejim.bbw.shims.IWorldShim;
+
+public class ArchitectureCraftCustomMapping extends CustomMapping {
+
+    private static final Block SHAPE_BLOCK = GameRegistry.findBlock("ArchitectureCraft", "shape");
```

Comment:
Please note that GTNH version of ArchitectureCraft has 2 different shape blocks: https://github.com/GTNewHorizons/ArchitectureCraft/blob/master/src/main/java/gcewing/architecture/ArchitectureContent.java#L73
Make sure to handle second block too (if one does exist).